easybuild/easyconfigs/n/NAMD/NAMD-2.12-intel-2017b-mpi.eb

```diff
@@ -0,0 +1,24 @@
+name = 'NAMD'
+version = '2.12'
+versionsuffix = '-mpi'
+
+homepage = 'http://www.ks.uiuc.edu/Research/namd/'
+description = """NAMD is a parallel molecular dynamics code designed for high-performance simulation of
+ large biomolecular systems."""
+
+toolchain = {'name': 'intel', 'version': '2017b'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+sources = ['NAMD_%(version)s_Source.tar.gz']
```

Comment:
@tovrstra Please include a SHA256 checksum here too (you can use `--inject-checksums` for that).